### ISSUE TITLE
[Translation] 8_5_threadpool.md

### ISSUE THREAD

**Issue by Johnny4Fun** [cfsamson/book-exploring-async-basics#43]
`most often the file I/O will be `Ready` almost immediately, so waiting for that in a event queue has very little effect in practice.`

I guess you mean waiting in a `thread pool` instead of `event queue`.

**Comment by cfsamson** [comment]
No, I meant that registering an interest to wait for a `Ready` event i an event queue does not have any positive effect since a lot of the operations will be `Ready` immediately. I actually go in to a bit more detail in this here: https://cfsamson.github.io/book-exploring-async-basics/9_1_file_module.html#bonus-material
